storybookjs/storybook
# Update install docs

## What I did

- Emphasize requirement to install in existing project
- Add link to Troubleshooting section
- Update welcome screenshot to include correct "edit" filename
- Re-arrange content order, so that Troubleshooting isn't last
- Update React Storybook examples
    - Remove react-native-web, as that project no longer uses Storybook
    - Point to SB Design System instead of react-dates, as it uses modern features
    
### Before

![InstallPage_before](https://user-images.githubusercontent.com/486540/135143612-e744802b-a589-4499-979e-c5cc2ae7da9d.png)

### After

![InstallPage_after](https://user-images.githubusercontent.com/486540/135143956-e4d82a6d-8855-4df0-956f-1fcebc536077.png)
_Note: The screenshot is **not** updated in this preview, because I could not get my locally running docs site to respect the updated image._ 🤷‍♂️ 

## How to test

1. Open the install page in the deploy preview
1. Check the changes listed above, for each framework
1. Confirm changes to `/app/react/README.md`


If your answer is yes to any of these, please make sure to include it in your PR.
